<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>五子棋游戏</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>五子棋游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_PVE">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>人机对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_PVP">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>双人对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Load">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读取游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Ranking">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>排行榜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Exit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
